src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
Here's what I imagined in c-code:
```
census_context* ctx = census_context_create(NULL, tags, ntags, status)
grpc_call* call = grpc_channel_create_call(channel, NULL, flags, completion_queue, method, host, deadline, NULL)
grpc_census_call_set_context(call, ctx)
....
```
Which seems like it could be done from the client side.